docs/RELEASE.md

```diff
@@ -1,21 +1,27 @@
 # Spice.ai Release Process
 
-## Release environments
+Spice.ai releases once per week.
 
-|                     | Production                     | Development                 | Local                         |
-| ------------------- | ------------------------------ | --------------------------- | ----------------------------- |
-| Short name          | prod                           | dev                         | local                         |
-| spiced Docker image | ghcr.io/spiceai/spiceai:latest | ghcr.io/spiceai/spiceai:dev | ghcr.io/spiceai/spiceai:local |
+## Branching Policy
+
+- At the beginning of the milestone for a target release a release branch is created, i.e. `release-v0.14.0-alpha`, in the main spiceai repository (not in a fork).
+- PRs are merged to the trunk branch incrementally as work is completed on individual features/bugs.
+- Once a feature/bugfix is considered "done" and ready to be included in the release, a new branch is created from the current release branch, all commits related to the feature/bugfix are cherry-picked into the new branch. And then a PR is created from the new branch into the release branch. That PR will be a regular merge, not a squash merge (IMPORTANT).
+- Any commits made directly to a release branch should be merged to the trunk branch.
+- If the decision to re-number the release is made by the release DRI, a new release branch with the correct version number is made from the existing release branch. The existing release branch is then deleted.
+- The engineer responsible for a given feature/bugfix is responsible for ensuring that the feature/bugfix is included in the release branch.
+
```

Comment:
Makes sense to add that release DRI is responsible for including external contributions into the release

Reply:
That should be part of the endgame, not the branching policy

Reply:
Added to endgame issue